src/Calculator/App.xaml

```diff
@@ -2179,7 +2179,27 @@
                                                                   AcceptsReturn="false"
                                                                   InputScope="Text"
                                                                   MaxLength="2048"
-                                                                  TextWrapping="NoWrap"/>
+                                                                  TextWrapping="NoWrap">
+                                            <Controls:MathRichEditBox.ContextFlyout>
+                                                <MenuFlyout x:Name="MathRichEditContextMenu">
```

Comment:
Do we need to add access/shortcut keys?

Reply:
We don't have these defined yet.  That said, we should add them, but likely as a follow up to this.